FOS.Models/Entities/GetUserTranslanderModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FOS.Models.Entities
+{
+    public class GetUserTranslanderModel
+    {
+        /// <summary>
+        ///CompanyId.
+        /// </summary>
+        [JsonProperty(PropertyName = "companyId", NullValueHandling = NullValueHandling.Ignore, Required = Required.Default)]
+        public int? CompanyId { get; set; }
+
+
+
+        /// <summary>
+        /// UserCode.
+        /// </summary>
+        [JsonProperty(PropertyName = "userCode", NullValueHandling = NullValueHandling.Ignore, Required = Required.Default)]
+        public string? UserCode { get; set; }
+
+        /// <summary>
+        /// User_ID.
+        /// </summary>
+        [JsonProperty(PropertyName = "userID", NullValueHandling = NullValueHandling.Ignore, Required = Required.Default)]
```

Comment:
It should be userId. Let us follow camel case so that it would be easy in the front end side to follow.